src/store/common/reducer.ts

```diff
@@ -3,9 +3,7 @@ import { CommonState, Mode } from './types';
 import { toggleAnnotationModeAction } from './actions';
 
 const modeReducer = createReducer<CommonState['mode']>(Mode.NONE, builder =>
-    builder.addCase(toggleAnnotationModeAction, (state, { payload: mode }: { payload: Mode }) =>
-        state === mode ? Mode.NONE : mode,
-    ),
+    builder.addCase(toggleAnnotationModeAction, (state, { payload: mode }: { payload: Mode }) => mode),
```

Comment:
This seems a bit risky, since the Preview SDK and Annotations SDK may not be released at the same time. Should we create a `setAnnotationMode` event or method, instead?

Reply:
Preview SDK changes are behind the discoverability feature flip, so Annotations SDK will release first, which is not risky, because all the usages in Preview SDK is already using `toggleAnnotationMode` as `setAnnotationMode`.